cruise/CruiseInformation.csv

```diff
@@ -64,16 +80,19 @@ PS1610A,R/V Pacific Storm,2016-10-11T16:00:00,2016-10-12T02:00:00,Endurance CE02
 NBP16-09,R/V Nathaniel B. Palmer,2016-10-21T10:35:00,2016-11-15T19:00:00,Argentine Basin 3
 NBP16-10,R/V Nathaniel B. Palmer,2016-11-20T11:14:00,2016-12-16T09:55:00,Southern Ocean 3
 PS1612,R/V Pacific Storm,2016-12-15T00:00:00,2016-12-15T00:00:00,Endurance Gliders
+TS20161219,F/V Tenacious II,2016-12-19T00:00:00,2016-12-19T23:59:59,Pioneer glider cp_336 recovery
 PS1701A,R/V Pacific Storm,2017-01-05T00:15:00,2017-01-07T00:30:00,Endurance CE02SHSP and Gliders
 P7a,M/V Warren Jr.,2017-01-15T00:00:00,2017-01-19T23:59:59,Pioneer 7a #TO BE DELETED
 WJ20170115,M/V Warren Jr.,2017-01-15T00:00:00,2017-01-19T23:59:59,Pioneer 7a
 EK-1701,R/V Elakha,2017-01-31T18:30:00,2017-01-31T22:40:00,Endurance GL info
 PS1702A,R/V Pacific Storm,2017-02-13T17:29:00,2017-02-13T23:20:00,deployed CE02SHSP-00005 and Glider 381
+TS20170306,F/V Tenacious II,2017-03-06T00:00:00,2017-03-06T23:59:00,Pioneer glider cp_335 recovery
 DS17-03A,F/V Discovery,2017-03-08T07:05:00,2017-03-08T14:00:00,Glider recovery.

```

Comment:
EA glider recovery 319

Reply:
Hmm?

Reply:
Never mind, not needed. Just happened to notice that one and knew which glider got recovered

Reply:
Copy that